### DISCUSSION TITLE
How to use "Change All Occurences"?

### DISCUSSION THREAD

**Discussion by bdrewery** [vscode-neovim/vscode-neovim#2311] [General]
I hilight a word, invoke the "change all occurrences" feature, it selects them all. Then what? I expected `cw` followed by the new word to replace them all but it did not. What is the proper way?

**Comment by theol0403** [comment]
Two ways:

1. Enter insert mode first and use vscode native method
2. Define a binding that uses into_insert (iirc that's what it's called) and the calls the command (there is a snippet for this in the readme)
